client/src/app/primitives/DropdownButton.less

```diff
@@ -0,0 +1,61 @@
+:local(.DropdownButton) {

```

Comment:
Are we able to re-use or mixin the existing `Button` styles? They already declare color, hover states and borders.

Reply:
Will fix. I'm not super familiar with the syntax yet. 😞